library/kotlin/src/io/envoyproxy/envoymobile/filters/FilterDataStatus.kt

```diff
@@ -44,5 +44,15 @@ sealed class FilterDataStatus {
    * This may be called by filters which must parse a larger block of the incoming data before
    * continuing processing, and will handle their own buffering.
    */
-  class StopIterationNoBuffer : FilterDataStatus()
+  class StopIterationNoBuffer<T : Headers> : FilterDataStatus<T>()
+
+  /**
+   * Resume previously-stopped iteration, possibly forwarding headers, if iteration was previously
+   * stopped during an on*Headers invocation.
```

Comment:
```suggestion
   * Resume previously-stopped iteration, possibly forwarding headers, if iteration was previously
   * stopped during an on*Headers invocation. If headers are already sent, headers should not be passed in the return status.

Reply:
`   * stopped during an on*Headers invocation. If headers have already been sent, headers should not be passed in the return status.`